tools/shaderc/shaderc_glsl.cpp

```diff
@@ -69,12 +61,26 @@ namespace bgfx { namespace glsl
 
 		const char* optimizedShader = glslopt_get_output(shader);
 
+		std::string out;
+		// Preserver #version preamble
+		if('#' == *optimizedShader)
+		{
+			bx::StringView preamble(optimizedShader, bx::strFindNl(optimizedShader).getPtr());
+			if(bx::strCmp(preamble, "#version", 8) == 0)
+			{
+				out.append(preamble.getPtr(), preamble.getLength());
+			}
+		}
+
```

Comment:
This will also break internal logic of patching shaders. If there is version, shader won't be patched at all in runtime, causing issues on some GL renderers.